content/en/docs/aws/deploy/install-kubeflow.md

```diff
@@ -204,30 +217,41 @@ istio-system   istio-ingress   *       a743484b-istiosystem-istio-2af2-xxxxxx.us
 
 This deployment may take 3-5 minutes to become ready. Verify that the address works by opening it in your preferred Internet browser.
 
+If you're using Basic Authentication, The credentials are the ones you specified in the KfDef file, or the default (`admin@kubeflow.org`:`12341234`). It is highly recommended to change the default credentials. To add static users or change the existing one, [add static users for basic auth](/docs/aws/deploy/install-kubeflow/#add-static-users-for-basic-auth).
 
-If you are using {{% config-uri-aws-standard %}}, the Kubeflow Dashboard can be accessed via istio-ingressgateway service.
-
-You can run following command to port forward to local, then open `http://localhost:8080` in browser.
-
-{{< highlight bash >}}
-kubectl port-forward svc/istio-ingressgateway -n istio-system 8080:80
-{{< /highlight >}}
+To secure enterprise level installation, use {{% config-uri-aws-cognito %}} and follow the [instructions](/docs/aws/authentication) to add authentication and authorization.
 
-Check more details [Ingress Gateway guide](https://istio.io/docs/tasks/traffic-management/ingress/ingress-control/)
+### Add static users for basic auth 
+To add users to basic auth, you just have to edit the Dex ConfigMap under the key staticPasswords.
 
-To expose Kubeflow with a LoadBalancer Service, just change the type of the `istio-ingressgateway` Service to `LoadBalancer`.
```

Comment:
Any reason to delete this?

You still need a way to expose service? either nodePort or port-forward, right? anything I am missing here?

Reply:
So, from my understanding, the reason we want to expose service: allow external traffic to access the Kubeflow Dashboard.

Since we enable ingress for ALB in Cognito and basic-auth. We don't need to expose Kubeflow with a LB service anymore for this purpose.

So I wonder if there is any reason that we still want to expose service? Or do I misunderstand the original reason we want to expose service?

Reply:
I thought you use nodeport yesterday. double checked it's ALB, then we can consolidate into one.